radix-engine/src/blueprints/pool/multi_resource_pool/blueprint.rs

```diff
@@ -232,7 +238,10 @@ impl MultiResourcePoolBlueprint {
             resource_bucket_amount_mapping
         };
 
-        let pool_unit_total_supply = substate.pool_unit_resource_manager.total_supply(api)?;
+        let pool_unit_total_supply = substate
+            .pool_unit_resource_manager
+            .total_supply(api)?
+            .unwrap();
```

Comment:
Suggest replacing all the  `.total_supply().unwrap()` with `.expect("Total supply is always enabled for pool unit resource")`